### ISSUE TITLE
saving full model?

### ISSUE THREAD

**Issue by bernardo-cruz** [devmount/GermanWordEmbeddings#33]
Great work you have done here in this repo - THX a lot. Would you mind saving your model as a full model in order to able continue training with new words?

**Comment by devmount** [comment]
Good point, I'll see when I can manage that (won't be in the next couple of weeks though). Still the model I trained now is nearly 8 years old, so I would nevertheless recommend training from scratch.
